### DISCUSSION TITLE
What is the meaning of a single and double "check" mark in VS Code in down right corner in the Status bar?

### DISCUSSION THREAD

**Discussion by DawidArdelli** [prettier/prettier-vscode#2959] [Q&A]
Hi, does anyone know what is the meaning of this mark? I've been searching for a while for an answer but I haven't found anywhere.

[screen](https://i.stack.imgur.com/J4xRY.png)

**Comment by zhouyc98** [comment]
It represents the document's formatting status.   
As a SO answer said (https://stackoverflow.com/questions/67187505/what-is-the-meaning-of-the-double-v-markedprettier-on-the-vscode):

> A Single Check represents that Prettier is enabled and working in your current environment and double-check symbol represents that your current document is "fine" according to the rules specified in the Prettier Extension.
